osbs/conf.py

```diff
@@ -272,6 +272,9 @@ def get_registry_api_versions(self, platform=None):
                                 "registry_api_versions",
                                 default='v1,v2')
         versions = value.split(',')
+        if not platform:
```

Comment:
osbs.conf at the client side, e.g. the Koji builder, will have registry_api_versions, yes. To support v1 it needs to be:

* osbs.conf on Koji builder says registry_api_versions=v1,v2
* client-config-secret says registry_api_versions=v1,v2 *and* platform section has enable_v1=true